static/sass/styles.scss

```diff
@@ -511,10 +511,143 @@ html {
   position: relative;
 }
 
+@media (width < $breakpoint-navigation-threshold) {
```

Comment:
Are the navigation changes relevant here? I guess these are from previous work by @alvaromateo ?
So would this be some rebase/merge issue to see them added here?

Reply:
Looks like these styles have been moved to another file, so this looks like a leftover from a rebase that can be removed.

https://github.com/canonical/snapcraft.io/pull/5348